deeposlandia/datasets/aerial.py

```diff
@@ -0,0 +1,144 @@
+"""Aerial dataset module
+
+Model Aerial images for convolutional neural network applications. Data is
+downloadable at https://project.inria.fr/aerialimagelabeling/files/.
+
+"""
+
+from multiprocessing import Pool
+import numpy as np
+import os
+from PIL import Image
+
+from deeposlandia.datasets import dataset
+from deeposlandia import utils
+
+
+class AerialDataset(dataset.Dataset):
+    """Dataset structure inspired from AerialImageDataset, a dataset released
+    by INRIA
+
+    The dataset is freely available at:
+    https://project.inria.fr/aerialimagelabeling/files/
+
+    It is composed of 180 training images and 180 testing images of size
+    5000*5000. There are ground-truth labels only for the former set.
+
+    Attributes
+    ----------
+    tile_size : int
+        Size of the tiles into which each raw images is decomposed during
+    dataset population (height=width)
+
```

Comment:
remaining useless new line in the docstring

Reply:
Done.